database/hall2operations.py

```diff
@@ -333,10 +333,7 @@ def _translation(self, N):
         return T
 
     def _change_of_basis_symbol(self, V):
-        if V[0] == '-':
-            return np.array([0, 0, -1])
-        else:
-            return np.array([0, 0, 1])
```

Comment:
@atztogo Why does the original implementation ignore the value of `V`?
For example, this function returns `[0, 0, 1]` for hall symbol `"P 62 2 (0 0 4) "`.

Reply:
Because the original implementation is compatible with the original spg.csv, which only has (0 0 1) and (0 0 -1).

Reply:
There is actually no need to keep compatibility with old `spg.csv` and such. These are all version controlled together

Reply:
@LecrisUT I think lan496 was asking why the old version `hall2operations.py` cannot handle something like `np.array([0, 0, 4])`, and I was explaining that the old version does not need to handle this. The PR does not deliberately keep the compatibility. Now the new `spg.csv` contains both, so the code need the ability to handle both.

Reply:
@lan496, my memory has completely gone about Hall symbols. But I think the answer by @site-g is explaining it.

Reply:
I see. Thank you for your explanations!